src/sonic-yang-models/yang-models/sonic-telemetry.yang

```diff
@@ -74,5 +74,26 @@ module sonic-telemetry {
             }
 
         }
+
+        container TELEMETRY_CLIENT_CERT {
+            description "TELEMETRY client cert list";
+
+            list TELEMETRY_CLIENT_CERT_LIST {
+                max-elements 8;
+                key "cert_cname";
+
+                leaf cert_cname {
+                    type string;
+                    description
+                        "client cert common name";
+                }
+
+                leaf role {
+                    type string;
+                    description
+                        "role of client cert common name";
+                }
+            }
+        }
```

Comment:
Please update unit test for sonic-yang-models

Reply:
Fixed